tools/appliance/builtin/template.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
do we need a license on this file?

Reply:
packer fails if you add a license. I've added reference as a comment instead.